guide/yaml/example_yaml/vanilla-bash-netcat-env.yaml

```diff
@@ -0,0 +1,12 @@
+name: Netcat Example with Environment Vars
+location: localhost
+services:
+- type: org.apache.brooklyn.entity.software.base.VanillaSoftwareProcess
+  name: Simple Netcat Server
+  launch.command: |
+    echo $MESSAGE | nc -l $NETCAT_PORT &
+    echo $! > $PID_FILE
+    
+  env:
+    MESSAGE: hello
```

Comment:
maybe worth adding a $brooklyn.config(....) in as the value, to emphasise the reusability mentioned in the .md, here and in the port.